src/mlpack/methods/ann/layer/bilinear_interpolation.hpp

```diff
@@ -118,6 +118,12 @@ class BilinearInterpolation
   //! Modify the depth of the input.
   size_t& InDepth() { return depth; }
 
+  //! Get the shape of the input.
+  size_t WeightSize() const
+  {
+    return InRowSize;
+  }
```

Comment:
Should this be `InputShape()`?  Also I think this won't compile; maybe it should be `InRowSize()` or just `inRowSize`?

Reply:
You are correct. It should be `InputShape()` and `inRowSize`